shop/views.py

```diff
@@ -425,3 +427,80 @@ def deletecomment(request, id):
     cmt = get_object_or_404(Comment, id=id).delete()
 
     return HttpResponseRedirect(request.META["HTTP_REFERER"])
+
+
+@staff_member_required
+def all_order_manage(request):
+    orders_w = Order.objects.filter(status = STATUS_ORDER['waitting'])
+    orders_p = Order.objects.filter(status = STATUS_ORDER['paid'])
+    
+    sum = 0
+    date = datetime.today()
```

Comment:
đặt tên biến rõ ràng nha
```suggestion
    current_date = datetime.today()
```